scripts/vagrant/zeppelin-dev/README.md

```diff
@@ -0,0 +1,154 @@
+
+Cloning the project again may seem counter intuitive, since this script likley originated from the project repository.  Consider copying just the vagrant/zeppelin-dev script from the zeppelin project as a stand alone directory, then once again clone the specific branch you wish to build.
+
+Synced folders enable Vagrant to sync a folder on the host machine to the guest machine, allowing you to continue working on your project's files on your host machine, but use the resources in the guest machine to compile or run your project.[^1]
+
+By default, Vagrant will share your project directory (the directory with the Vagrantfile) to `/vagrant`.  Which means you should be able to build within the guest machine after you   
+`cd /vagrant/incubator-zeppelin`
+
+
+### What's in this VM?
+
+Run the following commands, and should see the expected versions:
+
+`node --version` should report *v0.12.7*  
+`mvn --version` should report *Apache Maven 3.3.3* and *Java version: 1.7.0_85*
+
+
+The virtual machine consists of:
+
+ - Ubuntu Server 14.04 LTS
+ - Node.js 0.12.7
+ - npm 2.11.3
+ - Maven 3.3.3
+ - Git
+ - Unzip
+ - libfontconfig to avoid phatomJs missing dependency issues
+ - openjdk-7-jdk
+ - Python addons: pip, matplotlib, scipy, numpy, pandas
+ - Changes are made to the python backend configuration so that pyspark can render matplotlib plots without an interactive window
+ 
+ 
+### How to build & run Zeppelin
+
+This assumes you've already cloned the project either on the host machine in the zeppelin-dev directory (to be shared with the guest machine) or cloned directly into a directory while running inside the guest machine.
+
+```
+cd /incubator-zeppelin
+mvn clean package -Pspark-1.5 -Ppyspark -Dhadoop.version=2.2.0 -Phadoop-2.2 -DskipTests
+./bin/zeppelin-daemon.sh start
+```
+
+On your host machine browse to `http://localhost:8080/`
+
+If you [turned off port forwarding](#tweakvm) in the `Vagrantfile` browse to `http://192.168.51.52:8080`
+
+
+### [Tweaking the Virtual Machine](id:tweakvm)
+
+If you plan to run this virtual machine along side other Vagrant images, you may wish to bind the virtual machine to a specific IP address, and not use port fowarding from your local host.
+
+Comment out the "forward_port" line, and uncomment the private_network line in Vagrantfile, such as:
+
+```
+#config.vm.network "forwarded_port", guest: 8080, host: 8080
+config.vm.network "private_network", ip: "192.168.51.52"
+```
+
+`vagrant halt` followed by `vagrant up` will restart the guest machine bound to the IP address of 192.168.51.52.  
+This approach usually is typically required if running other virtual machines that discover each other directly by IP address, such as Spark Masters and Slaves as well as Cassandra Nodes, Elasticsearch Nodes, and other Spark data sources.  You may wish to launch nodes in virtual machines with IP Addresses such as: 192.168.51.53, 192.168.51.54, 192.168.51.53, etc..
```

Comment:
doesn't the ip range depends on the user's network config? we might want to mention that the subnet might vary and might not be 192.168.*